libc-test/build.rs

```diff
@@ -3457,6 +3457,8 @@ fn test_linux(target: &str) {
         "linux/netlink.h",
         // FIXME: requires Linux >= 5.6:
         [!musl]: "linux/openat2.h",
+        "linux/ptp_clock.h",
+        // FIXME: requires Linux >= 5.6:
```

Comment:
What is this referring to? uapi `ptp_clock.h` has been around since at least 4.0, not sure if this is intended for `ptrace.h` below but that doesn't make sense either.

Reply:
https://github.com/rust-lang/libc/pull/3559#issuecomment-1913615212 goes into why. it's not that the file doesn't exist before 5.6, but some of the constants are only added in this later version, so for the full functionality, 5.6 is required. I believe the same is true for `netlink.h` above, a file that is also very old but its api changed over time, so that is why I copied over that comment too

Reply:
I was asking mostly about why the `FIXME` is below `ptp_clock.h` rather than above, looks like it is referring to `ptrace.h` rather than `ptp_clock.h` - but I guess this didn't show up well on the review UI. That context is probably helpful though, maybe change the comment to `some fields require Linux >= 5.6`